web_interface/blueprints/api_v3.py

```
@@ -2822,6 +2840,22 @@ def uninstall_plugin():
+                # "config-entry-with-no-files" state and resurrect the
+                # plugin the user just asked to delete.
+                if hasattr(api_v3.plugin_store_manager, 'mark_recently_uninstalled'):
+                    api_v3.plugin_store_manager.mark_recently_uninstalled(plugin_id)
+
+                # Clean up plugin configuration BEFORE removing files so the
+                # config entry and the on-disk files disappear together from
+                # the reconciler's point of view.
+                if not preserve_config:
+                    try:
+                        api_v3.config_manager.cleanup_plugin_config(plugin_id, remove_secrets=True)
+                    except Exception as cleanup_err:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)